integ-test/src/test/java/org/opensearch/sql/legacy/ObjectFieldSelectIT.java

```diff
@@ -85,22 +85,6 @@ public void testSelectNestedFieldItself() {
     );
   }
 
-  @Test
```

Comment:
We can fix these rather than deleting them.
```
verifyDataRows(response, rows(new JSONArray(List.of(new JSONObject("{\"id\":1}"), new JSONObject("{\"id\":2}")))));
verifyDataRows(response, rows(new JSONArray(List.of(1,2))));
```